src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,63 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-block: var(--sbb-spacing-fixed-3x);
+
+  position: relative;
+  width: 100%;
+  overflow: hidden;
```

Comment:
Otherwise badge is shown this way:
![image](https://user-images.githubusercontent.com/101575400/189917197-c19810b4-5a45-46ac-ab62-4063dfa3f028.png)

Reply:
thanks, I've missed that :)